.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -9,13 +9,17 @@ inputs:
     default: "false"
     required: false
 
-  image-name:
-    description: The name of the image, should match the repository name in ECR
-    required: true
-
-  ecr-registry:
-    description: The ECR registry to push to, used in docker/login-action and for tagging images
-    default: public.ecr.aws/chainlink
+  ecr-prefix:
```

Comment:
How do you feel about having the name of `ecr-hostname`? I think that more accurately describes this variable

Reply:
Yeah that's a much better name, updated code and env